django/conf/urls/__init__.py

```diff
@@ -19,12 +19,29 @@
 def include(arg, namespace=None, app_name=None):
     if app_name and not namespace:
         raise ValueError('Must specify a namespace if specifying app_name.')
+    if app_name:
+        warnings.warn(
+            'The app_name argument to django.conf.urls.include() has been deprecated. '
+            'Set the app_name attribute in the included urlconf instead. ',
```

Comment:
URLconf
is "attribute" the correct term? I think of it as a "module-level variable."